services/server/src/models/game/route.js

```diff
@@ -18,6 +18,7 @@ router.get('/',
       skip: Joi.number().integer().min(0).default(0),
       creator: Joi.objectId().optional(),
       searchText: Joi.string().optional(),
+      customURL: Joi.string().optional(),
```

Comment:
If we transform customURL here then we can avoid a `pre('save')` hook at the DB level which might make things confusing

Reply:
I'm thinking we should have the `pre('save')` regardless, because the website is just one client of the API, there still could be others hitting the API directly.